daphne_worker/src/lib.rs

```diff
@@ -698,6 +698,7 @@ pub(crate) struct InternalTestVdaf {
     typ: String,
     #[serde(skip_serializing_if = "Option::is_none")]
     bits: Option<String>,
```

Comment:
Otherwise this field is mandatory.

```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
    bits: Option<String>,
```